File: c++/arrayReverse.cpp

#include "pch.h"
#include <iostream>

using namespace std; 

int main()
{
	int array[10] = { 7, 1, 2, 3, 4, 5, 6, 7 };
	reverse(begin(array), end(array)); 
	for (int i = 0; i < 10; i++) {
		cout << array[i]; 
	}
}

File: c#/Arrayreverse.cs
using System;
using System.Linq;

namespace ConsoleApp9
{
    class Program
    {
        static void Main(string[] args)
        {
            int[] array = { 7, 1, 2, 3, 4, 5, 6, 7 };
            int[] result = array.Distinct().ToArray();
            Array.Reverse(result);
            foreach (int value in result)
            {
                Console.WriteLine(value);
            }
            Console.WriteLine();
        }
    }
}
